feed_integration/app/src/main/java/tv/fw/feedintegration/recyclerview/FeedListFragment.kt

```diff
@@ -0,0 +1,84 @@
+package tv.fw.feedintegration.recyclerview
+
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import java.util.UUID
+import tv.fw.common.feed.FeedResource
+import tv.fw.error.FwError
+import tv.fw.error.FwErrorListener
+import tv.fw.feedintegration.BuildConfig.FW_CHANNEL_ID
+import tv.fw.feedintegration.BuildConfig.FW_PLAYLIST_ID
+import tv.fw.feedintegration.R
+import tv.fw.videofeed.FeedItemClickListener
+import tv.fw.videofeed.FeedViewState
+import tv.fw.videofeed.FeedViewStateListener
+import tv.fw.videofeed.baseOptions
+import tv.fw.videofeed.viewOptions
+
+/**
+ * A fragment representing a list of Items.
+ */
+class FeedListFragment : Fragment(), FeedItemClickListener, FwErrorListener, FeedViewStateListener {
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?,
+    ): View? {
+        return inflater.inflate(R.layout.fragment_feed_list, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        val recyclerView = view.findViewById<RecyclerView>(R.id.rvList)
+        recyclerView.layoutManager = LinearLayoutManager(context)
+        recyclerView.adapter = FeedListAdapter(getListItems(), lifecycle, this, this, this)
+    }
+
+    private fun getListItems(): List<ListItem> {
+        val playlistFeedViewOptions = viewOptions {
+            baseOptions {
+                supportBackwardAds(true)
+                feedResource(FeedResource.Playlist(FW_CHANNEL_ID, FW_PLAYLIST_ID))
+            }
+        }
+        return listOf(
+            TextItem(LOREN_IPSUM),
+            TextItem(LOREN_IPSUM),
+            FeedViewItem(UUID.randomUUID().toString(), playlistFeedViewOptions),
```

Comment:
I am wondering why not more feed views in the recycler view. 😈

Reply:
Because there is issue with it

Reply:
Ah! do we have a bug ticket for it?

Reply:
It is tricky, and needs investigation
No ticket yet